README.md

```diff
@@ -737,6 +738,24 @@ e.g:
 turbo crypto-fund --value 0.0001 --token kyve --private-key 'b27...45c'
 ```
 
+##### `upload-folder`
+
+Upload a folder of files to the Turbo Upload Service. This command will upload all files in the folder and create a manifest file for the folder.
```

Comment:
`and create a manifest file for the folder` could use a bit more specificity. Where does it "create" it? Is it uploaded? Can it be retrieved by the user? Programmatically?

Reply:
it uploads the manifest as a data item to turbo. it can be retrieved at arweave.net/manifest_tx_id